reference/test/reorder/mc64_kernels.cpp

```diff
@@ -284,6 +286,7 @@ TYPED_TEST(Mc64, CreatesCorrectPermutationAndScalingExampleSum)
         gko::experimental::reorder::Mc64<value_type, index_type>::build()
             .with_strategy(
                 gko::experimental::reorder::mc64_strategy::max_diagonal_sum)
+            .with_tolerance(real_type{1e-4})
```

Comment:
IMO, the default parameter of MC64 should be changed to depend on the value type instead.

Reply:
This still needs to be addressed.

Reply:
Ah, I tend to keep this in the test because it does not affect the test result with single/double precision.
Or, do you mean the default parameter of MC64?

Reply:
Yes, the default parameter in `include/ginkgo/core/reorder/mc64.hpp`. The current default will round to zero for half, so we should replace it.

Reply:
I pick 50*epsilon which is close to 1e-14 for double precision

Reply:
sounds reasonable.